netdcli/netdcli.go

```diff
@@ -182,10 +186,18 @@ func init() {
 		"cfg",
 		false,
 		"JSON file describing the global and network intent. Use '-' to read configuration from stdin")
+	flagSet.StringVar(&opts.stateStore,
+		"state-store",
+		utils.EtcdNameStr,
+		"State store to use")
 	flagSet.StringVar(&opts.etcdURL,
 		"etcd-url",
 		"http://127.0.0.1:4001",
 		"Etcd cluster url")
+	flagSet.StringVar(&opts.consulURL,
+		"consul-url",
+		"http://127.0.0.1:8500",
+		"Consul cluster url")
 	flagSet.StringVar(&opts.tenant,
```

Comment:
I kept these as separate flags since I wasn't sure if same url convention will apply to all data-stores that we integrate with. Also having per state-store configuration flag allowed me to have different **defaults** for the urls based on the `-state-store` that user specifies, so user can just run netdcli as `netdcli -state-store etcd -cfg <>` to use a etcd state-store with it's default url.